### DISCUSSION TITLE
Some photos are treated as motion photos, even though they don't appear to be such

### DISCUSSION THREAD

**Discussion by oryjkov** [immich-app/immich#9995] [Q&A]
### The bug

I've noticed that a few photos in my collection are treated as if they were motion photos (they get a little play icon on the thumbnail and get their own transcode) even though they don't seem to contain any "motion" part to them and immich server complains about not able to transcode/serve these.

I'm including two image files at the end of this post (github insists on showing the images full size).

1. First file is a file that looks like a pixel phone motion picture, but is not. Google photos also does not consider it a motion photo.
2. Second file is an actual motion photo taken from [this exiftool forum post](https://exiftool.org/forum/index.php?topic=14382.0)

I looked at what `exiftool` thinks about these files and it is what's tripping up immich [metadata.ts](https://github.com/immich-app/immich/blob/ce985ef8f8caf13f7f753c2d378fde75ea89f699/server/src/services/metadata.service.ts#L373)

```
# Both files claim to have a motion photo in them,

$ exiftool -s -struct -Directory PXL_20210721_090550242.MP.jpg PXL_20230118_124418216.MP.jpg
======== PXL_20210721_090550242.MP.jpg
Directory                       : [{Item={Length=0,Mime=image/jpeg,Padding=0,Semantic=Primary}},{Item={Length=6718630,Mime=video/mp4,Padding=0,Semantic=MotionPhoto}}]
======== PXL_20230118_124418216.MP.jpg
Directory                       : [{Item={Length=0,Mime=image/jpeg,Padding=0,Semantic=Primary}},{Item={Length=4334107,Mime=video/mp4,Padding=0,Semantic=MotionPhoto}}]
    2 image files read
# But only one files contains the MP4 header
$ grep -P '\x66\x74\x79\x70\x69\x73\x6F\x6D' PXL_20210721_090550242.MP.jpg PXL_20230118_124418216.MP.jpg
grep: PXL_20230118_124418216.MP.jpg: binary file matches
$
```

1. ![PXL_20210721_090550242 MP](https://github.com/immich-app/immich/assets/23466884/b9d89565-7472-48b7-9587-e64bac7b5f5c)
2. ![PXL_20230118_124418216 MP](https://github.com/immich-app/immich/assets/23466884/3913a7a7-4d54-4e2c-a757-1dbe1148b03b)

### The OS that Immich Server is running on

container

### Version of Immich Server

v1.105.1

### Version of Immich Mobile App

v1.105.0

### Platform with the issue

- [x] Server
- [ ] Web
- [ ] Mobile

### Your docker-compose.yml content

```YAML
not relevant
```

### Your .env content

```Shell
not relevant
```

### Reproduction steps

```bash
1. Upload the two files in the report
2. Immich treats both as motion photos and extracts what it thinks should be an mp4 file
3. Immich fails to transcode mp4 for one of the files
...
```

### Relevant log output

```shell
[Nest] 2  - 06/05/2024, 2:23:13 PM   ERROR [ImmichServer] [ExceptionsHandler] ENOENT: no such file or directory, access 'upload/encoded-video/4f13d54e-b06a-48dc-8f7e-1d47fffe1425/6e/72/6e728ff0-452
e-48f5-acf3-3323c98f9c3c.mp4'
Error: ENOENT: no such file or directory, access 'upload/encoded-video/4f13d54e-b06a-48dc-8f7e-1d47fffe1425/6e/72/6e728ff0-452e-48f5-acf3-3323c98f9c3c.mp4'
    at async access (node:internal/fs/promises:603:10)
    at async sendFile (/usr/src/app/dist/utils/file.js:51:9)
    at async AssetControllerV1.serveFile (/usr/src/app/dist/controllers/asset-v1.controller.js:54:9)
```

**Comment by jrasm91** [comment]
Where is your first photo from? If it has metadata for a motion photo, and is called `*.MP.jpg*` which follows the motion photo naming pattern why isn't it a motion photo? Has the file been modified by some other system, tool, or script that removed the embedded video?
